extensions/wikia/ExactTargetUpdates/client/builders/BaseRequestBuilder.class.php

```diff
@@ -50,6 +54,16 @@ public function withGroup( $group ) {
 		return $this;
 	}
 
+	public function withWikiCategories( $categories ) {
+		$this->wikiCategories = $categories;
+		return $this;
+	}
+
+	public function withWikiId( $wikiId ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename "$wikiId" which has the same name as the field declared at line 26. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=php%3AS1117)